packages/mobile/src/account/Profile.test.tsx

```diff
@@ -2,30 +2,35 @@ import * as React from 'react'
 import 'react-native'
 import { fireEvent, render } from 'react-native-testing-library'
 import { Provider } from 'react-redux'
+import { setName } from 'src/account/actions'
 import Profile from 'src/account/Profile'
-import { navigate } from 'src/navigator/NavigationService'
 import { Screens } from 'src/navigator/Screens'
+import { RootState } from 'src/redux/reducers'
 import { createMockStore, getMockStackScreenProps } from 'test/utils'
 
-function profileFactory() {
-  return (
-    <Provider store={createMockStore({})}>
-      <Profile {...getMockStackScreenProps(Screens.Profile)} />
-    </Provider>
-  )
-}
-
 describe('Profile', () => {
   it('renders correctly', () => {
-    const { toJSON } = render(profileFactory())
+    const store = createMockStore({})
```

Comment:
Two nits: no need to pass in `{}` and no need to remake the store multiple times (line 23 below). The tests can share this value